src/main/java/com/cvsgo/exception/GlobalExceptionHandler.java

```diff
@@ -87,9 +80,10 @@ public ErrorResponse handleIOException(IOException e) {
     }
 
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+    @ExceptionHandler(Exception.class)
     public ErrorResponse handleException(Exception e) {
         log.error("예기치 못한 예외가 발생했습니다.", e);
-        return ErrorResponse.of(ErrorCode.UNEXPECTED_ERROR.name(), e.getMessage());
+        return ErrorResponse.of(ErrorCode.UNEXPECTED_ERROR.name(), ErrorCode.UNEXPECTED_ERROR.getMessage());
```

Comment:
from 메소드를 이용하면 좋을 것 같아요!
```suggestion
        return ErrorResponse.from(ErrorCode.UNEXPECTED_ERROR);
```